### PR TITLE
Code changes for getting all session details

### PR DESCRIPTION
**Pull request by jayandran-Sampath**
## Change list

Please provide briefly described change list which are you going to propose. 
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted java code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: src/test/java/io/appium/java_client/android/AndroidDriverTest.java [hunk 1]
```diff
@@ -43,276 +43,281 @@
 
 public class AndroidDriverTest extends BaseAndroidTest {
 
-    @Test
```
Comment (SrinivasanTarget) [inline src/test/java/io/appium/java_client/android/AndroidDriverTest.java]: Can you not alter the indentation and list only added test?

File: src/main/java/io/appium/java_client/HasSessionDetails.java [hunk 2]
```diff
@@ -17,15 +17,20 @@
 package io.appium.java_client;
 
 import static io.appium.java_client.MobileCommand.GET_SESSION;
+import static io.appium.java_client.MobileCommand.GET_ALLSESSION;
```
Comment (SrinivasanTarget) [inline src/main/java/io/appium/java_client/HasSessionDetails.java]: Checkstyle will fail here. GET_ALLSESSION should go first.

File: src/main/java/io/appium/java_client/HasSessionDetails.java [hunk 3]
```diff
@@ -17,15 +17,20 @@
...
 import static java.util.Optional.ofNullable;
 import static java.util.stream.Collectors.toMap;
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
+import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 
 import org.openqa.selenium.remote.Response;
 
+import java.util.Arrays;
```
Comment (SrinivasanTarget) [inline src/main/java/io/appium/java_client/HasSessionDetails.java]: Looks like an unused import.

File: src/main/java/io/appium/java_client/MobileCommand.java [hunk 4]
```diff
@@ -271,6 +273,8 @@
         commandRepository.put(TOGGLE_DATA, postC("/session/:sessionId/appium/device/toggle_data"));
         commandRepository.put(COMPARE_IMAGES, postC("/session/:sessionId/appium/compare_images"));
         commandRepository.put(EXECUTE_DRIVER_SCRIPT, postC("/session/:sessionId/appium/execute_driver"));
+        //Get All Session Details
```
Comment (SrinivasanTarget) [inline src/main/java/io/appium/java_client/MobileCommand.java]: comments not required.

File: src/test/java/io/appium/java_client/android/AndroidDriverTest.java (hunk 1 above)
Comment (jayandran-Sampath) [reply src/test/java/io/appium/java_client/android/AndroidDriverTest.java]: @SrinivasanTarget sure, reverted the file and added my test

File: src/main/java/io/appium/java_client/HasSessionDetails.java (hunk 2 above)
Comment (jayandran-Sampath) [reply src/main/java/io/appium/java_client/HasSessionDetails.java]: @SrinivasanTarget  will update, thanks

File: src/main/java/io/appium/java_client/HasSessionDetails.java (hunk 3 above)
Comment (jayandran-Sampath) [reply src/main/java/io/appium/java_client/HasSessionDetails.java]: yes, removed that line @SrinivasanTarget

File: src/main/java/io/appium/java_client/HasSessionDetails.java [hunk 5]
```diff
@@ -86,4 +89,15 @@ default boolean isBrowser() {
         return ofNullable(getSessionDetail("browserName"))
                 .orElse(null) != null;
     }
+    
+    /**
+     * Get All Sessions details
+     * @return List of Map objects with All Session Details
```
Comment (SrinivasanTarget) [inline src/main/java/io/appium/java_client/HasSessionDetails.java]: Add period at EOL.

**Pull request merged**